webapp-django/crashstats/crashstats/middleware.py

```diff
@@ -14,11 +14,18 @@ class SetRemoteAddrFromForwardedFor(object):
     anybody can "fake" their IP address. Only use this when you can absolutely
     trust the value of HTTP_X_FORWARDED_FOR.
     """
+    def __init__(self, get_response=None):
+        self.get_response = get_response
```

Comment:
This redoes our middlewares to support the new middleware API. The new API has some differences in structure and also supports a shortcut property. I thought about using the middleware mixin, but fixing our middlewares seemed pretty straight-forward, so I did that instead.

https://docs.djangoproject.com/en/2.0/topics/http/middleware/#upgrading-pre-django-1-10-style-middleware